projects/mac_ai/testing/prepare_mac_ai.py

```
@@ -280,5 +280,58 @@ def cleanup_gguf_models(base_work_dir):
         logging.info(f"{dest} does not exists, nothing to remove.")
         return
 
-    logging.info(f"Removing {dest} ...")
-    remote_access.run_with_ansible_ssh_conf(base_work_dir, f"rm -r {dest}")
+    # Get the current model(s) to preserve them
+    current_model = config.project.get_config("test.model.name")
+    current_models_fnames = []
+
+    # `test.model.name` can be a string or a list; normalize to a list.
+    if current_model == None:
```

Ruff (0.14.8)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)